test/clj/game/cards/events_test.clj

```diff
@@ -2460,12 +2461,12 @@
       (is (= 7 (count (:discard (get-runner)))) "7 cards in discard")
       (is (= 5 (count (:deck (get-runner)))) "5 cards shuffled back into stack")
       (is (= 1 (count (:rfg (get-runner)))) "HART removed from game")))
-  (testing "Cannot play with empty heap"
-    (do-game
-      (new-game {:runner {:hand ["Harmony AR Therapy"]}})
-      (take-credits state :corp)
-      (play-from-hand state :runner "Harmony AR Therapy")
-      (is (empty? (:prompt (get-runner))) "HART was not played")))
+  ;(testing "Can play with empty heap"
+  ;  (do-game
+  ;    (new-game {:runner {:hand ["Harmony AR Therapy"]}})
+  ;    (take-credits state :corp)
+  ;    (play-from-hand state :runner "Harmony AR Therapy")
+  ;    (is (= 2 (count (prompt-buttons :runner))) "No Cards to Shuffle")))
```

Comment:
Did you mean to remove this test?

Reply:
older commit, got fixed but the file is currently in a conflicted merge. thanks!